src/standard_json.rs

```diff
@@ -0,0 +1,47 @@
+//! This module defines the json format for `solang compile --standard-json`.
```

Comment:
Please add the SPDX header: `// SPDX-License-Identifier: Apache-2.0`

Reply:
Good point, fixed!